project/ISSUES.md

```diff
@@ -0,0 +1,179 @@
+
+* `area/API`
+* `area/cleanup`
+* `area/jobs`
+* `area/tests`
+* `area/GIEs`
+* `area/toolshed`
+* `area/UI-UX`
+* `area/workflows`
+
+This list will definitely grow over time.
+
+Other Useful Labels
+-------------------
+
+While the three labels sets indicating kind, status, and area are required
+there are several other labels that are be useful and/or have special purpose.
+
+* `procedures` is a special tag that indicates that the issue is related to
+  project governance, and it overrides the need for the trio of
+  kind/status/area tags, and these are never auto-flagged for triage.
+
+* `planning` is also a special tag that indicates the issue is related to
+  larger-scale issue planning.  These issues are typically meta-issues
+  containing checklists and references to other issues which are subcomponents
+  and stepping stones necessary for issue resolution.  These *can* utilize the
+  `focus/*` tags but are not required to.  Status and type make little sense
+  here.
+
+* `roadmap` is a reserved tag for the primary project roadmap.  This is a
+  meta-issue that is not expected to be completed, but rather serves as an
+  entry point to the high level development of the project.
+
+* `beginner-friendly` can be used to indicate a nice entry-level issue that
+  only requires limited understanding of the larger Galaxy framework and
+  ecosystem.  This is useful for encouraging new contributors.
+
+
+The Roadmap
+===========
+
+We will maintain a single `Roadmap` tagged meta-issue which will describe (at a
+very high level) the *current* major areas of focus for the project.  This is
+similar to our PRIORITIES 2014/15 cards on Trello.  Using [Task
+Lists](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments),
+this issue will link to sub-issues which will go into much more detail, might
+have its own checklists to even more subcomponent cards, and so on.  
+
+This `Roadmap` issue will be assigned to every release milestone, forcing
+periodic review of the roadmap.
+
+To prevent the roadmap from being tied completely to GitHub, and to facilitate
+portable change tracking over time, we will also maintain the file
+project/ROADMAP.md within the repository.  Whenever the ROADMAP issue text is
+changed, ROADMAP.md should be updated correspondingly.
+
+This document won't have the organizational integration that a live GitHub
+issue does, but this way we're be able to have a ROADMAP.md permanently
+attached to the code regardless of what issue tracking or organizational
+software we use in the future.
+
+
+Voting
+======
+
+Users can vote for issues by commenting with a +1.  It's possible to sort the
+issue list by 'most commented' which would be a good indicator of what issues
+are 'hot', though this doesn't necessarily indicate a high vote.  It's possible
+that that this is good enough good enough and in some ways potentially more
+useful to find 'hot' issues than a flat vote count.
+
```

Comment:
100% do not do hotness - that was a joke. Maybe ``popular/5``, ``popular/10``, ``popular/15``, ``popular/controversial``.